packages/block-editor/src/components/font-sizes/fluid-utils.js

```diff
@@ -133,17 +192,20 @@ export function getComputedFluidTypographyValue( {
 		3
 	);
 
-	const viewPortWidthOffset = minViewPortWidthOffsetValue + fontSizeUnit;
-	let linearFactor =
+	const viewPortWidthOffset =
+		roundToPrecision( minViewPortWidthOffsetValue, 3 ) + fontSizeUnit;
+	const linearFactor =
 		100 *
 		( ( maximumFontSizeParsed.value - minimumFontSizeParsed.value ) /
 			( maximumViewPortWidthParsed.value -
 				minumumViewPortWidthParsed.value ) );
-	linearFactor = roundToPrecision( linearFactor, 3 ) || 1;
```

Comment:
This used to have a fallback of `1`, I imagine to avoid multiplying by `0`. Can the above calculation still theoretically result in `0`? E.g. if the max font size value is the same as the minimum font size value, or if the max viewport width is the same as the minimum viewport width?

E.g, if I set `min` and `max` to the same value in a preset (not very useful, I know), then the output will result in a `* 0`:

<img width="418" alt="image" src="https://user-images.githubusercontent.com/14988353/196350076-df6f0689-2b7a-4af8-bbd3-bb6f713a6229.png">

In that case, prior to this PR the multiplier would have been `* 1`.

I think we can resolve this here by updating the below lines to:

```js
	const linearFactorScaled = roundToPrecision(
		( linearFactor || 1 ) * scaleFactor,
		3
	);
```

Reply:
Oh thanks. That nearly slipped through. 🤭 

I'll add a test as well

Reply:
Or, we could bail early when the min and max are the same. Having a clamped value in that case, as you hint at, is not very useful 😄